newsroom/wire/search.py

```diff
@@ -444,7 +444,30 @@ def get_matching_bookmarks(self, item_ids, active_users, active_companies):
         """
         bookmark_users = []
 
-        search_results = self.get_items(item_ids)
+        query = {
+            'bool': {
+                'must_not': [
+                    {'term': {'type': 'composite'}},
+                ],
+                'must': [
+                    {'terms': {'_id': item_ids}}
+                ],
+            }
+        }
+        user = get_user()
```

Comment:
`get_matching_bookmarks` is called from `notify_user_matches` in push.py. There won't be any `user` or `company`.  We don't need `set_product_query` or `set_bookmarks_query` functions as well.

Reply:
Done